src/metaschema/oscal_ssp_metaschema.xml

```diff
@@ -646,6 +647,11 @@
         <enum value="no">The component does not allow an authenticated scan.</enum>
       </allowed-values>
 
+      <allowed-values target="component/link/@rel" allow-other="yes">
+        &allowed-values-component_component_link-rel;
+        <enum value="imported-from">Identifies the UUID or URI the <code>component-definition</code> that originally defined the <code>component</code>.</enum>
+      </allowed-values>
+
```

Comment:
Since this targets the component directly, this constraint should be on `system-component`.